message PBMsgReqHeroItemList
{
  required int64 hid = 1;
}
